### ISSUE TITLE
Use with s3 compatible storage

### ISSUE THREAD

**Issue by snerdish** [andrewrk/node-s3-cli#14]
My s3cmd config file specifies a local radosgw for s3 compatible storage, but s3-cli doesn't take note of this.

**Comment by BartVB** [comment]
Same problem here. This is probably because s3-cli doesn't pickup the `host_base` and '`host_bucket` options from .s3cfg:
http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Endpoint.html
http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Config.html#s3BucketEndpoint-property

**Comment by deguich** [comment]
Yes, I've got the same problem. I use a bad solution, I hardcoded the endpoint to my rados gateway in source code.

In s3-cli/cli.js line 90:

```
s3Options: {
      accessKeyId: accessKeyId,
      secretAccessKey: secretAccessKey,
      sslEnabled: !args.insecure,
      endpoint: 'myradosgw.org',
      region: args.region,
    },
```

I can dev and "pull request" the solution If andrewk is OK.

**Comment by ghost** [comment]
Can this be pushed please? Being able to use compatible storage is important.
This fix still works and it would be great if the tool officially supports the endpoint input.
